filer/management/commands/orphaned_files.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+
+from django.core.files.storage import DefaultStorage
+from django.core.management.base import BaseCommand
+from django.utils.translation import ugettext_lazy as _
+
+from filer.settings import DEFAULT_FILER_STORAGES
+
+
+class Command(BaseCommand):
+    help = _("Look for orphaned files in media folders.")
+    storage = DefaultStorage()
+    prefix = DEFAULT_FILER_STORAGES['public']['main']['UPLOAD_TO_PREFIX']
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--delete',
+            action='store_true',
+            dest='delete',
+            default=False,
+            help=_("Delete found orphaned files in media folders."),
+        )
+
+    def handle(self, *args, **options):
+        from filer.models.filemodels import File
+
+        def walk(absdir, reldir):
+            child_dirs, files = self.storage.listdir(absdir)
+            for filename in files:
+                relfilename = os.path.join(reldir, filename)
+                try:
+                    File.objects.get(file=relfilename)
```

Comment:
why not `if not File.objects.filter(file=relfilename).exists()`

Reply:
fixed